### PR TITLE
Community Calendar blog post

### PR DESCRIPTION
**Pull request by 2color**
https://bafybeib2xqqhfbpm76b42iri7xbpxf4jsefaiygo6bwfjfbanvgpqhrdaa.on.fleek.co/2023-01-ipfs-community-calendar/

## TODO

- [x] Remove/replace the commit graph image
- [x] Add a short line for each WG/event in the calendar 
- [x] Address https://github.com/ipfs/ipfs-blog/pull/513#discussion_r1052442299
- [x] Add blurb and URL to COD WG
- [x] Add This month in IPFS livestream

### CODE REVIEW

File: src/_blog/2022-12-community-calendar.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Announcing the New IPFS Community Calendar
+description: 'Check out the new IPFS community calendar where you can participate and contribute to one of the many working groups advancing IPFS.'
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:3]:
```suggestion
description: 'Check out the new IPFS community calendar where you can you can discover and register for IPFS community events, working group meetings, and more.'
```
equal emphasis on newer & more integrated/WG communities

File: src/_blog/2022-12-community-calendar.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- [Move the Bytes Working Group](https://lu.ma/8kk9i628)
+- [IPFS Content Routing Working Group](https://lu.ma/ipfs-routing-wg)
+- [Compute over Data (COD) Working Group](TODO)
+- [IPVM Community Call](https://lu.ma/ipvm)
+- [IPFS Thing 2023](https://lu.ma/ipfsthing-preregistration)
+
+
+We couldn't be more excited to welcome you to join the IPFS events of your interest! 
+
+If you're interested in hosting an event — feel free to submit it for inclusion or [reach out to us](mailto:devrel@ipfs.tech) with any questions.
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:55]:
```suggestion
If you're interested in hosting an event, feel free to submit it for inclusion or [reach out to us](mailto:devrel@ipfs.tech) with any questions.
```

File: src/_blog/2022-12-community-calendar.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- [ProbeLab Office Hours](TODO)
+- [IPFS Implementers Sync](https://lu.ma/ipfs-implementers)
+- [Move the Bytes Working Group](https://lu.ma/8kk9i628)
+- [IPFS Content Routing Working Group](https://lu.ma/ipfs-routing-wg)
+- [Compute over Data (COD) Working Group](TODO)
+- [IPVM Community Call](https://lu.ma/ipvm)
+- [IPFS Thing 2023](https://lu.ma/ipfsthing-preregistration)
+
+
+We couldn't be more excited to welcome you to join the IPFS events of your interest!
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:53]:
```suggestion
We couldn't be more excited to welcome you to join the IPFS events that interest you! 
```

File: src/_blog/2022-12-community-calendar.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+This shift is evident on several fronts: the sheer number number of working groups regularly meeting, the scope and breadth of new projects (expanding IPFS into compute with [[Bacalhau](https://www.bacalhau.org/)), and activity in the [IPFS specifications repo](https://github.com/ipfs/specs/) which has spiked in the last year.
+
+![Contributions to the IPFS Specs](../assets/ipfs-calendar/ipfs-specs-github-contributors.png)
+
+## The New IPFS Community Calendar
+
+Today, we're excited to share the new [IPFS Community Calendar](https://lu.ma/ipfs) as a hub for all IPFS community events — both offline and online.
+
+The new calendar should better serve both newcomers navigating their journey into the IPFS community, and longstanding members becoming aware of other working groups.
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:28]:
```suggestion
The new calendar should better serve both newcomers navigating their journey into the IPFS community, and longstanding members becoming aware of other working groups.
```

This overlaps a bit with line 39 and is a bit tricky to parse for newcomers. Maybe remove altogether?

File: src/_blog/2022-12-community-calendar.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+The community calendar should serve as the event hub for all these different groups. We already have events covering a wide spectrum of levels and areas of interest:
+
+- [InterPlantary Office Hours](https://lu.ma/IP-Office-Hours)
+- [ProbeLab Office Hours](TODO)
+- [IPFS Implementers Sync](https://lu.ma/ipfs-implementers)
+- [Move the Bytes Working Group](https://lu.ma/8kk9i628)
+- [IPFS Content Routing Working Group](https://lu.ma/ipfs-routing-wg)
+- [Compute over Data (COD) Working Group](TODO)
+- [IPVM Community Call](https://lu.ma/ipvm)
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:49]:
```suggestion
- [IPVM Community Call](https://lu.ma/ipvm) - focused on implementers in the effort to add content-addressed computation to IPFS
```

File: src/_blog/2022-12-community-calendar.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+🎫 If you're organising a meetup or event related to IPFS, submit it for inclusion using the "Submit Event" button on [the calendar](https://lu.ma/ipfs). Once submitted we'll review it and publish it (assuming it's relevant.)
+
+
+## Events for everyone
+
+IPFS has many facets as a modular suite of distributed protocols for representing, routing, and transferring content-addressed data. The versatility of IPFS attracts many group such as application developers, low-level protocol developers, service providers, and artists and creatives.
+
+The community calendar should serve as the event hub for all these different groups. We already have events covering a wide spectrum of levels and areas of interest:
+
+- [InterPlantary Office Hours](https://lu.ma/IP-Office-Hours)
```
Comment (mishmosh) [inline src/_blog/2022-12-community-calendar.md:43]:
```suggestion
- [InterPlantary Office Hours](https://lu.ma/IP-Office-Hours) - a weekly meetup open to all, where you can bring your IPFS questions and meet others
```

feel free to edit the specifics, but let's add a short blurb about each to invite engagement

**Review by mishmosh — APPROVED** [review]

**Comment by mishmosh** [comment]
The commits graph is a bit confusing. Is that from the specs repo? Since this post is about the Community Calendar, the image should probably be a screenshot or animated gif related to the calendar — or we can omit an image entirely.

**Comment by 2color** [comment]
> The commits graph is a bit confusing. Is that from the specs repo? 

It is — I added it with the intention of the alt text showing under it, but I forgot that the blog doesn't render alt text below the image. 

> Since this post is about the Community Calendar, the image should probably be a screenshot or animated gif related to the calendar — or we can omit an image entirely.

I'll give it a try and see how it looks

**Pull request merged**